Screens/SnapInfoScreen.tsx

```diff
@@ -5,41 +5,122 @@ import { NavigationContainer } from '@react-navigation/native';
 import { createNativeStackNavigator } from '@react-navigation/native-stack';
 import Banner from '../Components/Banner';
 
-function SnapInfoScreen({navigation}) {
+function SnapInfoScreen({ navigation }) {
 
-return (
+  return (
     <View style={styles.mainContainer}>
-      <Banner/>
-        <Text style={styles.snapInfoText}>
-            Snap Information
+      <Banner />
+      <Text style={styles.snapInfoText}>
+        Snap Information
+      </Text>
+
+      <View style={styles.subContainer}>
+        <Text style={styles.subContainerTitle}>
+          ELIGIBILITY
+        </Text>
+        <Text style={styles.subContainerText}>
+          Lorem ipsum dolor sit amet consectetur.
+        </Text>
+      </View>
+
+      <View style={styles.subContainer}>
+        <Text style={styles.subContainerTitle}>
+          WHERE TO APPLY
+        </Text>
+        <Text style={styles.subContainerText}>
+          Lorem ipsum dolor sit amet consectetur.
         </Text>
+      </View>
```

Comment:
NIT: Honestly this is just a me thing, but this looks like a reusable component we can maybe store in `./Components`, where we can just code up a component that takes in title and body text as props. I don't really see any more of these components in the design though, this is more of a personal preference in case we do end up using more of these containers. Up to you if you want to add this.

Reply:
I think I'll rewrite it when it comes up.